uniffi_macros/src/lib.rs

```diff
@@ -369,14 +369,12 @@ pub fn generate_and_include_scaffolding(udl_file: TokenStream) -> TokenStream {
     }.into()
 }
 
-/// A dummy macro that does nothing.
-///
-/// This exists so `#[uniffi::export]` can emit its input verbatim without
-/// causing unexpected errors, plus some extra code in case everything is okay.
-///
-/// It is important for `#[uniffi::export]` to not raise unexpected errors if it
-/// fails to parse the input as this happens very often when the proc-macro is
-/// run on an incomplete input by rust-analyzer while the developer is typing.
+/// An attribute for constructors.
+/// Constructors are in `impl` blocks which have a `#[uniffi::export]` attribute.
+/// Because there's no `derive` for the block, it's impossible to have
+/// `#[uniffi(constructor)]`.
```

Comment:
This is not true actually. It could be `#[uniffi(constructor)]`, but then the macro output of `#[uniffi::export]` would be invalid when it fails to parse the input and re-emits verbatim it for IDE compatibility.